--- HRP4C.urdf	2013-11-20 10:33:32.408793666 +0900
+++ HRP4C.dev.urdf	2013-11-20 02:08:16.047412199 +0900
@@ -442,7 +442,7 @@
     <inertial>
       <mass value="0.3735" />
       <origin xyz="0.021 0 0" rpy="0 -0 0"/>
-      <inertia ixx="0" ixy="0" ixz="0" iyy="0.00016471" iyz="0" izz="0.00016471"/>
+      <inertia ixx="0.00016471" ixy="0" ixz="0" iyy="0.00016471" iyz="0" izz="0.00016471"/>
     </inertial>
   </link>
   <gazebo reference="EYE_P_LINK">
@@ -465,7 +465,7 @@
     <inertial>
       <mass value="0.3735" />
       <origin xyz="0.021 0 0" rpy="0 -0 0"/>
-      <inertia ixx="0" ixy="0" ixz="0" iyy="0.00016471" iyz="0" izz="0.00016471"/>
+      <inertia ixx="0.00016471" ixy="0" ixz="0" iyy="0.00016471" iyz="0" izz="0.00016471"/>
     </inertial>
   </link>
   <gazebo reference="EYE_Y_LINK">
@@ -1145,7 +1145,7 @@
   <gazebo reference="L_WRIST_R">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="L_HAND_J0" type="revolute">
+  <joint name="L_HAND_J0" type="fixed">
     <parent link="L_WRIST_R_LINK"/>
     <child  link="L_HAND_J0_LINK"/>
     <origin xyz="0.025 0.0046 -0.0526" rpy="0 -0 0 "/>
@@ -1161,7 +1161,7 @@
   <gazebo reference="L_HAND_J0">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="L_HAND_J1" type="revolute">
+  <joint name="L_HAND_J1" type="fixed">
     <parent link="L_WRIST_R_LINK"/>
     <child  link="L_HAND_J1_LINK"/>
     <origin xyz="-0.009 0 -0.0959" rpy="0 -0 0 "/>
@@ -1225,7 +1225,7 @@
   <gazebo reference="NECK_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="CHEEK_P" type="revolute">
+  <joint name="CHEEK_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="CHEEK_P_LINK"/>
     <origin xyz="-0.0265164 0 0.0255761" rpy="0 -0 0 "/>
@@ -1241,7 +1241,7 @@
   <gazebo reference="CHEEK_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="EYEBROW_P" type="revolute">
+  <joint name="EYEBROW_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="EYEBROW_P_LINK"/>
     <origin xyz="-0.0357313 0 0.0900806" rpy="0 -0 0 "/>
@@ -1257,7 +1257,7 @@
   <gazebo reference="EYEBROW_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="EYELID_P" type="revolute">
+  <joint name="EYELID_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="EYELID_P_LINK"/>
     <origin xyz="0.082 0.0315 0.0596149" rpy="0 -0 0 "/>
@@ -1273,7 +1273,7 @@
   <gazebo reference="EYELID_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="EYE_P" type="revolute">
+  <joint name="EYE_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="EYE_P_LINK"/>
     <origin xyz="0.0816179 0.0315 0.0596149" rpy="0 -0 0 "/>
@@ -1289,7 +1289,7 @@
   <gazebo reference="EYE_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="EYE_Y" type="revolute">
+  <joint name="EYE_Y" type="fixed">
     <parent link="EYE_P_LINK"/>
     <child  link="EYE_Y_LINK"/>
     <origin xyz="0 0 0" rpy="0 -0 0 "/>
@@ -1305,7 +1305,7 @@
   <gazebo reference="EYE_Y">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="MOUTH_P" type="revolute">
+  <joint name="MOUTH_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="MOUTH_P_LINK"/>
     <origin xyz="0.0389284 0 0.00507761" rpy="0 -0 0 "/>
@@ -1321,7 +1321,7 @@
   <gazebo reference="MOUTH_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="LOWERLIP_P" type="revolute">
+  <joint name="LOWERLIP_P" type="fixed">
     <parent link="MOUTH_P_LINK"/>
     <child  link="LOWERLIP_P_LINK"/>
     <origin xyz="0.0573582 0 -0.0274567" rpy="0 -0 0 "/>
@@ -1337,7 +1337,7 @@
   <gazebo reference="LOWERLIP_P">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="UPPERLIP_P" type="revolute">
+  <joint name="UPPERLIP_P" type="fixed">
     <parent link="NECK_P_LINK"/>
     <child  link="UPPERLIP_P_LINK"/>
     <origin xyz="0.102 0 0.012" rpy="0 -0 0 "/>
@@ -1449,7 +1449,7 @@
   <gazebo reference="R_WRIST_R">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="R_HAND_J0" type="revolute">
+  <joint name="R_HAND_J0" type="fixed">
     <parent link="R_WRIST_R_LINK"/>
     <child  link="R_HAND_J0_LINK"/>
     <origin xyz="0.025 -0.0046 -0.0526" rpy="0 -0 0 "/>
@@ -1465,7 +1465,7 @@
   <gazebo reference="R_HAND_J0">
     <cfmDamping>0.4</cfmDamping>
   </gazebo>
-  <joint name="R_HAND_J1" type="revolute">
+  <joint name="R_HAND_J1" type="fixed">
     <parent link="R_WRIST_R_LINK"/>
     <child  link="R_HAND_J1_LINK"/>
     <origin xyz="-0.009 0 -0.0959" rpy="0 -0 0 "/>
